docs/modules/ROOT/pages/advanced-how-tos/managing-compliance-with-ec/customizing-policy.adoc

```diff
@@ -0,0 +1,30 @@
+= Customizing Policy
+
+An Enterprise Contract Policy defines which checks are performed by EC during integration tests and
+release. If EC reports violations, the best course of action, in most cases, is to adjust the build
+process to mitigate the violation. For example, if EC reports that a certain required scanner was
```

Comment:
I feel like mitigate implies you're just making it a little less severe or painful. How about "eliminate" or remedy?
```suggestion
process to eliminate the violation. For example, if EC reports that a certain required scanner was
```

```suggestion
process to remedy the violation. For example, if EC reports that a certain required scanner was
```

Reply:
Went with remedy.